src/scheduler/job_info.c

```diff
@@ -3247,15 +3243,15 @@ find_jobs_to_preempt(status *policy, resource_resv *hjob, server_info *sinfo, in
 
 		po = schd_get_preempt_order(pjob);
 		if (po != NULL) {
-			if (po->order[0] == PREEMPT_METHOD_SUSPEND && pjob->job->can_suspend) {
+			if (policy->rel_on_susp != NULL && po->order[0] == PREEMPT_METHOD_SUSPEND && pjob->job->can_suspend) {
```

Comment:
You need a similar change in find_and_preempt_jobs() too. This is only happening in the simulated universe.